objectRecognitionSource/build.gradle

```diff
@@ -72,6 +72,7 @@ test {
     // set heap size for the test JVM(s)
     minHeapSize = "2048m"
     maxHeapSize = "2048m"
+    forkEvery = 1
```

Comment:
Combination of the way the tests are written (static vbles) and Gradle's test server causes issues in the OR tests.  This avoids them by limiting the re-use.  Can circle back and clean up some of the tests, probably, but wanted to get things going.